README.md

```diff
@@ -3390,6 +3390,16 @@ Other Style Guides
     - We primarily use [`mocha`](https://www.npmjs.com/package/mocha) at Airbnb. [`tape`](https://www.npmjs.com/package/tape) is also used occasionally for small, separate modules.
     - 100% test coverage is a good goal to strive for, even if it’s not always practical to reach it.
     - Whenever you fix a bug, _write a regression test_. A bug fixed without a regression test is almost certainly going to break again in the future.
+    
+  <a name="testing--is-documentation"></a><a name="30.3"></a>
+  - [30.3](#testing--is-documentation) **Is your documentation**:
+    - Tests should explain what your code does.
+    - First write a case for each main functionality, and then their exceptions.
+    - Refactor tests until they are easy to read and understand.
+    - Never use object internal representation from test code, use objects as other code will.
```

Comment:
Can you elaborate on this one?

Reply:
I rewrite it.

Reply:
> Never use _"private"_ properties to test code.

yes... I elaborate making it shorter O: ) 
I do not want to break the _"rhythm"_ of the all testing section.

I think that it is enough telling to anyone do not use "private" stuff, because they already know what means "private". And if not, and they search for "private" they will find more things later.
--


"_ of all testing section making something large
I do not want to break the _"rythm of all testing section making something largeI do not want to break the _"rythm"_ of all testing section making something large